app/src/main/java/org/p2p/wallet/home/model/TokenComparator.kt

```diff
@@ -4,18 +4,13 @@ import org.p2p.core.token.Token
 import org.p2p.core.utils.isMoreThan
 
 /**
- * First element should always be USDC || USDT
- * Further, we sort by total in USD
+ * We sort by total in USD
  * If there is no USD rate for the token, we sort by its total amount
  * Other tokens are not user's, no sorting is applied
  * */
 class TokenComparator : Comparator<Token> {
 
     override fun compare(o1: Token, o2: Token): Int = when {
-        o1.isUSDC -> -1
-        o2.isUSDC -> 1
-        o1.isUSDT && !o2.isUSDC -> -1
-        o2.isUSDT && !o2.isUSDC -> 1
         o1 is Token.Active && o2 is Token.Active -> compareActiveTokens(o1, o2)
```

Comment:
```suggestion
        o1 is Token.Active && o2 is Token.Active -> compareActiveTokens(o1, o2)
        
```
can be rewritten as  o1.totalInUsd?.compareTo(o2.totalInUsd)